apps/zigbeeTest.matrix/README.MD

```diff
@@ -0,0 +1,16 @@
+Events
```

Comment:
Do we want to include the **zigbeeTest** app? maybe we should just add it to [matrix-srcjs-apps](https://github.com/matrix-io/matrix-srcjs-apps)